modules/ibi-extra-partition-ibi-install-iso.adoc

```diff
@@ -0,0 +1,91 @@
+// Module included in the following assemblies:
+//
+// * edge_computing/ibi-image-based-install.adoc 
+
+:_mod-docs-content-type: PROCEDURE
+[id="ibi-extra-partition-ibi-install-iso_{context}"]
+= Configuring additional partitions on the target host
+
+The installation ISO creates a partition for the `/var/lib/containers` directory as part of the image-based installation process. 
+
+You can create additional partitions by using the `coreosInstallerArgs` specification. For example, in hard disks with adequate storage, you might need an additional partition for storage options, such as {lvms-first}. 
+
+[NOTE]
+====
+The `/var/lib/containers` partition requires at least 500 GB to ensure adequate disk space for precached images. You must create additional partitions with a starting position larger than the partition for `/var/lib/containers`.
```

Comment:
500G?
I was able to deploy where the entire drive was 200G

Reply:
It is quite large but this was the agreed recommended size. Some convo about it [here](https://redhat-internal.slack.com/archives/C05JHD9QYTC/p1733748765321229?thread_ts=1733747767.254319&cid=C05JHD9QYTC).